tests/osn-tests/src/test_osn_transition.ts

```diff
@@ -0,0 +1,150 @@
+    });
+
+    context('# types', () => {
+        it('Get all transition types', () => {
+            // Gettin all transition types
+            transitionTypes = osn.TransitionFactory.types();
+
+            // Checking if transitionTypes array contains the basic obs transition types
+            expect(transitionTypes.length).to.not.equal(0);
+            expect(transitionTypes).to.include.members(basicOBSTransitionTypes);
+        });
+    });
+
+    context('# create', () => {
+        it('Create all transition types', () => {
+            let transitionType: string;
+            let transition: ITransition;
+            const transitionName: string = 'test_osn_transition_create';
+            // Create each transition type available
+            for (transitionType of transitionTypes)
+            {
+                transition = osn.TransitionFactory.create(transitionType, transitionName);
+
+                // Checking if transition was created correctly
+                expect(transition).to.not.equal(undefined);
+                expect(transition.id).to.equal(transitionType);
+                expect(transition.name).to.equal(transitionName);
+                transition.release();
+            }
+        });
+
+        it('Create all transition types with settings', () => {
+            let transitionType: string;
+            let transition: ITransition;
+            let settings: ISettings = {};
+            settings['test'] = 1;
+            const transitionName: string = 'test_osn_transition_create_settings';
+
+            // Create each transition type availabe passing settings parameter
+            for (transitionType of transitionTypes)
+            {
+                transition = osn.TransitionFactory.create(transitionType, transitionName, settings);
+
+                // Checking if transition was created correctly
+                expect(transition).to.not.equal(undefined);
+                expect(transition.id).to.equal(transitionType);
+                expect(transition.name).to.equal(transitionName);
+                transition.release();
+            }
+        });
+
+    });
+
+    context('# createPrivate', () => {
+        it('Create all transition types', () => {
+            let transitionType: string;
+            let transition: ITransition;
+            const transitionName: string = 'test_osn_transition_create_private';
+
+            // Create each transition type available
+            for (transitionType of transitionTypes)
+            {
+                transition = osn.TransitionFactory.createPrivate(transitionType, transitionName);
+
+                // Checking if transition was created correctly
+                expect(transition).to.not.equal(undefined);
+                expect(transition.id).to.equal(transitionType);
+                expect(transition.name).to.equal(transitionName);
+                transition.release();
+            }
+        });
+
+       
+    });
+
+    context('# set', () => {

```

Comment:
Add getActiveSource and clear to the description of context here to inform that those functions are also being tested here